views/snippets_api.inc.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Langchecker;
+
+use Transvision\Json;
+
+// This view works only for snippets (website ID 6)
+$current_website = $sites[6];
+$current_locale = $locale;
+$json_data = [];
+
+if ($locale == '') {
+    exit('<p>Missing locale code in the request</p>');
```

Comment:
We should return a 400 error with the error message, as we do on Transvision, maybe add this static method to the Json class:
```php
public static function invalidAPICall($error)
{
    http_response_code(400);
    return self::output(['error' => $error], false, true)
}
```